src/Razor/src/Microsoft.VisualStudio.LanguageServer.ContainedLanguage/SimpleTextChange.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.VisualStudio.Text;
+
+namespace Microsoft.VisualStudio.LanguageServer.ContainedLanguage
+{
+    internal class SimpleTextChange : ITextChange
```

Comment:
Any reason to not just call it `VisualStudioTextChange`?

Reply:
Yeah, I'm surprised there isn't a public equivalent as well. 

I copied this from [https://devdiv.visualstudio.com/DevDiv/_git/VS-Platform?path=%2Fsrc%2FEditor%2FVisualStudio%2FImpl%2FUndo%2FSimpleTextChange.cs](https://devdiv.visualstudio.com/DevDiv/_git/VS-Platform?path=%2Fsrc%2FEditor%2FVisualStudio%2FImpl%2FUndo%2FSimpleTextChange.cs) and kept the name, but I don't have a problem with renaming this to VisualStudioTextChange as it certainly makes sense. 

---
In reply to: [447116718](https://github.com/dotnet/aspnetcore-tooling/pull/2137#discussion_r447116718) [](ancestors = 447116718)